howl/data/tokenize.py

```diff
@@ -49,7 +48,7 @@ def add_word(self, word: str):
     def max_split(self, tokens: Union[List[str], str]) -> Tuple[Union[List[str], str], Union[List[str], str]]:
         node = self.root
         counter = 0
-        for tok in tokens:
+        for tok in tokens.lower():
```

Comment:
Does `.lower()` work properly if tokens is a list?

Reply:
The notation was incorrect.
I think ralph initially thought about triggering this function from multiple places for different purposes.
However, this is called from only once place right now and takes in string

I have updated the types, if supporting List[str] is necessary in the future, I will add it back

Reply:
Ok great